doc/quickstart.md

```diff
@@ -0,0 +1,210 @@
+Welcome to
+# Pyccel
+
+ ## What is Pyccel
+
+-   Static compiler for Python 3, using Fortran or C as backend language.
+-   Started as small open-source project in 2018 at IPP Garching.
+-   Public repository is now hosted on GitHub, freely available for download.
+
+ ## Python’s objects, variables, and garbage collection
+
+ Python is an **interpreted** language, **dynamically typed** and **garbage-collected**.
+
+ ### Python object
+
+-   Is created by the Python interpreter when `object.__new__()` is invoked (e.g. as a result of an expression).
+-   Can be either mutable or immutable, but its type never changes.
+-   Resides in memory and has a **reference count**.
+-   Is accessed through one or more Python variables.
+-   Is destroyed by the garbage collector when its reference count drops to zero.
+
+For more details about Python object, see [this](https://docs.python.org/3/tutorial/classes.html).
+
+ ### Python variable
+
+-   Is a reference to a Python object in memory.
+
+-   Is created with an assignment operation `x = expr`:
+    -   If the variable `x` already exists, the interpreter reduces the reference count of its object
+    -   Otherwise a new variable `x` is created, which references the value of expr.
+    -   The reference count increased with an assignment operator, in argument passing or appending an object to a list.
```

Comment:
```
    -   If the variable `x` already exists, the interpreter reduces the reference count of its object
    -   Otherwise a new variable `x` is created
    -   The variable `x` is then modified to reference the object referenced by `expr` and the reference count of this object is increased
```